Library/Homebrew/cmd/search.rb

```diff
@@ -146,9 +146,21 @@ def search_names(query, string_or_regex, args)
     print_formulae &&= all_formulae.any?
     print_casks &&= all_casks.any?
 
-    ohai "Formulae", Formatter.columns(all_formulae) if print_formulae
+    if print_formulae
+      if $stdout.tty?
+        ohai "Formulae", Formatter.columns(all_formulae)
+      else
+        puts all_formulae
+      end
```

Comment:
I feel like this pattern is sufficiently common that we should have a separate method to handle it already.

Reply:
@carlocab For `ohai` specifically or just having conditional printing like this?

Reply:
Having conditional printing depending on whether `$stdout.tty?`, I mean.